rock-paper-scissors/scripts/rock-paper-scissors.js

```diff
@@ -0,0 +1,158 @@
+const gameChoices = ["rock", "paper", "scissors"];
+let playerSelection = "";
+let computerSelection = "";
+
+let playerWins = 0;
+let computerWins = 0;
+let nobodyWins = 0;
+let roundCount = 0;
+
+//initiates the new round by calling startNewRound. startNewRound calls for
+// the player and computer functions to make a selection.
+
+startNewRound();
+function startNewRound() {
+  playerPlay();
+  computerPlay();
+  playGame(playerSelection, computerSelection);
+}
+
+function playerPlay() {
+  playerSelection = prompt("Please enter Rock, Paper, or scissors").toLowerCase();
+  return playerSelection;
+}
+
+function computerPlay() {
+  computerSelection = gameChoices[Math.floor(Math.random() * gameChoices.length)];
+  return computerSelection;
+}
+// playGame takes the player and computer input. passes computerPlay argument to the correct
+//scoring function.
+
+function playGame(playerSelection, computerSelection) {
+  if (playerSelection === "rock") {
+    rock(computerSelection);
+  } else if (playerSelection === "paper") {
+    paper(computerSelection);
+  } else if (playerSelection === "scissors") {
+    scissors(computerSelection);
+  }
+}
+
+// "add" functions tally the score. called by functions rock(), paper(), scissors().
+function addPlayerWins(playerSelection, computerSelection) {
+  console.log(`You win! ${playerSelection} beats ${computerSelection}`);
+  roundCount += 1;
+  playerWins += 1;
+}
+function addComputerWins(playerSelection, computerSelection) {
+  console.log(`You Lose! ${computerSelection} beats ${playerSelection}!`);
+  roundCount += 1;
+  computerWins += 1;
+}
+function addNobodyWins() {
+  console.log(`This round was a draw! Try again!`);
+  roundCount += 1;
+  nobodyWins += 1;
```

Comment:
So the way I alluded to simplifying these three functions is by noticing `roundCount += 1` is a piece of code that is repeated, regardless of who won. That means we can likely extract it (like a mathematical equation) and put it outside. i.e. `xa + xb + xc == x(a + b + c)` 

If you move `roundCount += 1` to l68, l79, and l90. You can remove some of the duplication. The fact that it still needs to be in three places indicates we can simplify further, but I think it's good enough for now.